fe/projects/AspNetCore/src/areas/Products/components/ProductDetail/ProductDetail.scss

```diff
@@ -115,14 +148,59 @@
             border-bottom: 1px solid $swamp;
             padding: 1rem;
             width: 50%;
-        }
+        }        
+    }
+
+    .desktop-product-gallery {
+        display: none;
     }
 }
 
-.image-gallery-image {
-    min-height: 446px;
+.swiper {
+    padding-bottom: 20px;
+    margin-bottom: 10px;
+}
 
-    @media only screen and (max-width: $tablet) {
-        min-height: 300px;
+.swiper-button-next, .swiper-button-prev {
+    color: #000;    
+    background-color: #fff;
+    border-radius: 50%;
+    width: 2rem;
+    height: 2rem;
+}
+
+.swiper-button-next::after, .swiper-button-prev::after {
+    font-size: 0.5rem;
+    font-weight: bold;
+}
+
+@media only screen and (min-width: $desktop) {
```

Comment:
**suggestion (non-blocking):** You can move this to line: 154 for desktop-product-gallery